src/components/atom_preference_language_checkbox/index.tsx

```diff
@@ -0,0 +1,48 @@
+import { GlobalLanguageCode } from '@/global.interface'
+import { preferenceState } from '@/recoil/preferences/preference.state'
+import { Checkbox, FormControlLabel } from '@mui/material'
+import { FC, SyntheticEvent } from 'react'
+import { useRecoilCallback, useRecoilValue } from 'recoil'
+
+interface Props {
+  languageCode: GlobalLanguageCode
+}
+
+const PreferenceLanguageCheckbox: FC<Props> = ({ languageCode }) => {
+  const preference = useRecoilValue(preferenceState)
+  const onChange = useRecoilCallback(
+    ({ set, snapshot }) =>
+      async (e: SyntheticEvent<Element, Event>, checked: boolean) => {
+        const preference = await snapshot.getPromise(preferenceState)
+        if (!preference) return
+
+        const element = e.target as HTMLInputElement
+        const languageCode = element.value as GlobalLanguageCode
```

Comment:
`onChange` no longer requires`languageCode` from e.target.value
Can you fix it?

Reply:
This also means the ”value" is no longer required

Reply:
Handled in https://github.com/ajktown/wordnote/pull/101/commits/95abf3d4be1171c31f1f52f81ef6f0eadfdca03c